rails/db/schema.rb

```diff
@@ -405,13 +405,13 @@
     t.string   "description",                      limit: 255
     t.integer  "width",                            limit: 4
     t.integer  "height",                           limit: 4
-    t.string   "url",                              limit: 255
+    t.text     "url",                              limit: 65535
```

Comment:
I believe browsers generally have URL limits along the lines of ~4K, so 65K is probably excessive. Is there any advantage in this context to specifying a value like 4K or 8K, i.e. is there any database efficiency to be had by being more conservative?

Reply:
@kswenson It seems strange to me, but in trying to set a more reasonable limit I found there was no way to do so. It seems like if we're to use data type `text` (which makes sense since all the other types only allow a max of 255 or are even larger than 65535), you're stuck with the 65535 limit. You can't make it smaller. I may be wrong about that, but I haven't yet found anything to contradict that observation.

Reply:
That's why I asked it as a question. It may well be that it doesn't make any difference to the database implementation, so there's no reason to specify a smaller value. I was just curious about whether or not it mattered.